.gitignore

```diff
@@ -24,3 +24,5 @@ build/Release
 
 # Dependency directory
 node_modules
+
+.idea/
```

Comment:
do we need this ?

Reply:
it is for my JetBrains environment - normally I put it in my repos but wont put it into others